persistence/sql/persister_sessiontokenexchanger.go

```diff
@@ -0,0 +1,62 @@
+// Copyright © 2023 Ory Corp
+// SPDX-License-Identifier: Apache-2.0
+
+package sql
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/gofrs/uuid"
+
+	"github.com/ory/kratos/selfservice/sessiontokenexchange"
+	"github.com/ory/x/otelx"
+	"github.com/ory/x/sqlcon"
+)
+
+var _ sessiontokenexchange.Persister = new(Persister)
+
+func (p *Persister) CreateSessionTokenExchanger(ctx context.Context, flowID uuid.UUID, code string) (err error) {
+	ctx, span := p.r.Tracer(ctx).Tracer().Start(ctx, "persistence.sql.CreateSessionTokenExchanger")
+	defer otelx.End(span, &err)
+
+	e := sessiontokenexchange.Exchanger{
+		NID:    p.NetworkID(ctx),
+		FlowID: flowID,
+		Code:   code,
+	}
+
+	e.ID, err = uuid.NewV4()
+	if err != nil {
+		return err
+	}
```

Comment:
This will cause high latency on INSERTS on RBR tables. It's faster if the system (cockroach) generates these IDs

Reply:
fixed.